dora/tests/src/test/java/alluxio/client/fs/io/BufferedBlockInStreamIntegrationTest.java

```diff
@@ -36,7 +35,7 @@
 import java.util.List;
 
 /**
- * Integration tests for {@link BlockInStream}.
+ * Integration tests for {@link FileInStream}.
```

Comment:
This test is actually against the file API. So I'm actually keeping it here, despite the fact that it's marked `@Ignore` and to-be-fixed.